docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -19,165 +19,202 @@ This tutorial was last tested with [Mina zkApp CLI](https://github.com/o1-labs/z
+  ```ts src/index.ts
+  1 import { Square } from './Square.js';
+  2
+  3 export { Square };
+  ```
 
-:::info
+  The `src/index.ts` file contains all of the exports you want to make available for consumption from outside your smart contract project, such as from a UI.
 
-The following commands are included for reference, and will fail if you run them at this point. This is because we haven't written the `Square` smart contract yet.
+## Write the zkApp Smart Contract
 
-:::
+Now, the fun part! Write your smart contract in the `src/Square.ts` file. 
 
-To compile the TypeScript code into JavaScript, and run the JavaScript code, you'll type:
+Line numbers are provided for convenience. A final version of the smart contract is provided in the [Square.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/Square.ts) example file.
 
-```
-$ npm run build
-$ node build/src/main.js
-```
+This tutorial walks through the `Square` smart contract code in the `src/Square.ts` as viewed in a text editor. Because this is your first project, this tutorial uses the provided example file to ensure you enjoy the best user experience.  
 
-The first line creates JavaScript code in the build directory, while the second line runs the code in `src/main.ts`.
+## Copy the example 
 
-You can also combine these together into one line, such as:
+This tutorial walks you through the [Square.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/Square.ts) example file. 
 
-```
-$ npm run build && node build/src/main.js
-```
+1. First, open the [Square.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/Square.ts) example file.
 
-This will run `main` if the build is successful.
+1. Copy the entire contents of the file into your smart contract in the `src/Square.ts` file.
 
-## Write the zkApp Smart Contract
+Now you are ready to review the imports in the smart contract.  
 
-Now, the fun part! Let's write our smart contract: `src/Square.ts`. Line numbers are provided for convenience. A final version of what we're writing can be found [here](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/Square.ts).
+### Imports
 
-:::tip
+The `import` statement brings in other packages and dependencies to use in your smart contract. 
 
-This tutorial will walk through the `Square` smart contract linked above. We recommend that you copy and paste the entire file from that link, and then follow the tutorial with the finished code in place.
+:::info
 
-You can also copy and paste the code snippets into your project as you go - to avoid inserting the line numbers into your smart contract, copy these code snippets using the button provided. It will appear at the top right of the snippet box when you mouseover it.
+All functions used inside a smart contract must operate on SnarkyJS compatible data types: `Field` types and other types built on top of `Field` types. 
```

Comment:
@jasongitmail  can you help me learn what a Field element is? can we call it Field elements? always cap Field? or is it a Field type? or?